### PR TITLE
French opensource.md

### PR DESCRIPTION
**Pull request by debPhoenix**
Translate the changes from #411 into French! :)
Could you please label this PR with the label "hacktoberfest-accepted" before you merge it? :)
Thanks!

### CODE REVIEW

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the PR, @debPhoenix. However, I am not sure I see the changes from #411. There should be two new repositories and "yace" should be changed to "backend" and get a different description.

Also, please undo the tu -> vous changes. We specifically decided on the informal form to be in line with the German version. :)

**Comment by debPhoenix** [comment]
Hi! Can you tell me if all is right now?
Thanks you! :)

**Comment by baltpeter** [comment]
Thanks! From what I can see, the new repositories *media* and *letter-generator* are still missing. That's those:

https://github.com/datenanfragen/website/pull/411/files#diff-9c905a9578ef04eeae24fb44b0f72e64R48-R72

**Comment by debPhoenix** [comment]
Ok. I understand! Can you tell me if all is right now?

**Comment by baltpeter** [comment]
Looks good now. @NathanBnm Would you mind doing the review? Otherwise I'll do it.

File: content/fr/open-source.md [hunk 1]
```diff
@@ -72,35 +98,35 @@ Voici un aperçu des principaux dépôts que nous avons créés directement ou i
...
 <article class="list-article icon-list-article">
-    <div class="col25 article-featured-image"><a href="https://github.com/baltpeter/yace"><img class="image" src="/card-icons/speech-bubble.svg" alt="yace"></a></div>
+    <div class="col25 article-featured-image"><a href="https://github.com/datenanfragen/backend"><img class="image" src="/card-icons/speech-bubble.svg" alt="backend"></a></div>
     <div class="padded col75">
-        <a href="https://github.com/baltpeter/yace"><h1><code>yace</code></h1></a>
+        <a href="https://github.com/datenanfragen/backend"><h1><code>backend</code></h1></a>
         <span class="license">Licence MIT</span>
         <p class="description">
-            Un moteur simple, respectueux de la vie privée et facile à déployer pour créer des solutions de commentaires personnalisées, hébergé sur AWS.
+            Un serveur de développement qui définit les paramètres des fonctionnalités dynamiques du site internet comme les commentaires, les suggestions eet les donations.:;;
```
Comment (NathanBnm) [inline content/fr/open-source.md:106]: There is an extra letter to "et"

File: content/fr/open-source.md [hunk 2]
```diff
@@ -72,35 +98,35 @@ Voici un aperçu des principaux dépôts que nous avons créés directement ou i
...
     <div class="clearfix"></div>
-    <a class="button button-primary read-more-button" href="https://github.com/baltpeter/yace">Voir le dépôt&nbsp;<span class="icon icon-arrow-right"></span></a>
+    <a class="button button-primary read-more-button" href="https://github.com/datenanfragen/backend">Voir le dépôt&nbsp;<span class="icon icon-arrow-right"></span></a>
 </article>
 
 <a id="contributeurs"></a>
 ## Contributeurs
 
-Un grand merci à tous les contributeurs de demandetesdonnees.fr ! Le projet ne serait pas possible sans vous.
+Un grand merci à tous les contributeurs(trices) de demandetesdonnees.fr! Le projet ne serait pas possible sans vous.
```
Comment (NathanBnm) [inline content/fr/open-source.md:116]: There is a missing space before exclamation mark

**Review by NathanBnm — CHANGES_REQUESTED** [review]
Just a few things but it's fine 👍

File: content/fr/open-source.md (hunk 1 above)
Comment (debPhoenix) [reply content/fr/open-source.md]: done! :)

File: content/fr/open-source.md (hunk 2 above)
Comment (debPhoenix) [reply content/fr/open-source.md]: done! :)

**Review by NathanBnm — APPROVED** [review]

**Comment by debPhoenix** [comment]
Could you please label this PR with the label "hacktoberfest-accepted" before you merge it? :)

**Review by baltpeter — APPROVED** [review]
Thank you, @debPhoenix and @NathanBnm!

**Comment by baltpeter** [comment]
> Could you please label this PR with the label "hacktoberfest-accepted" before you merge it? :)

No need for that. As this repo is tagged with `hacktoberfest`, an accepting review or a merge is [enough for it to count](https://hacktoberfest.digitalocean.com/hacktoberfest-update). :)

**Pull request merged**